circuitbreaker/README.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
- [ ] pls add a link in the README in the root of the repository
- [ ] pls add a test workflow